### PR TITLE
Add management for the hosting-queued service

### PR DESCRIPTION
**Pull request by javierbertoli**

### CODE REVIEW

File: aegir/service.sls [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% from "aegir/map.jinja" import aegir with context %}
+
+aegir_hostmaster:
+  service.running:
+    - name: {{ aegir.hostmaster.service }}
+    - enable: true
+    - require:
+      - pkg: aegir
```
Comment (noelmcloughlin) [inline aegir/service.sls:11]: How will require be satisfied?  Are you missing include?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,10 @@
 aegir formula
 =============
 
+0.0.3 (2018-07-31)
+
+- Add service management for hosting-queued
```
Comment (noelmcloughlin) [inline CHANGELOG.rst:6]: Very good ;-)

File: aegir/service.sls (hunk 1 above)
Comment (javierbertoli) [reply aegir/service.sls:11]: You're right. Added.

Comment (noelmcloughlin) [reply aegir/service.sls:11]: Did you push it?

Comment (javierbertoli) [reply aegir/service.sls:11]: Yes, [line 3](https://github.com/saltstack-formulas/aegir-formula/pull/4/files#diff-34f2299e5a32f710c65673af57bac1c2R3) above :smile:

Comment (noelmcloughlin) [reply aegir/service.sls:11]: Sorry, maybe I never refreshed the page.

**Review by noelmcloughlin — APPROVED** [review]
LGTM thanks for your hard work @javierbertoli

**Pull request merged**